roles/kafka_connect_replicator/tasks/rbac_replicator_monitoring.yml

```diff
@@ -1,4 +1,16 @@
 ---
+- name: Get Authorization Token for Connect Replicator Monitoring Interceptor ERP User
+  import_role:
+    name: common
+    tasks_from: get_authorization_token.yml
+  vars:
+    oauth: "{{ oauth_enabled }}"
+    oauth_user: "{{ kafka_connect_replicator_monitoring_interceptor_erp_oauth_user }}"
+    oauth_password: "{{ kafka_connect_replicator_monitoring_interceptor_erp_oauth_password }}"
+    ldap_user: "{{ kafka_connect_replicator_monitoring_interceptor_erp_admin_user }}"
+    ldap_password: "{{ kafka_connect_replicator_monitoring_interceptor_erp_admin_password }}"
+  when: rbac_enabled or kafka_connect_oauth_enabled
```

Comment:
same as above, why do we have kafka_connect_oauth_enabled and oauth_enabled here?